content/en/docs/tutorials/stateful-application/basic-stateful-set.md

```diff
@@ -300,7 +322,15 @@ liveness and readiness, you can use the SRV records of the Pods (
 application will be able to discover the Pods' addresses when they transition
 to Running and Ready.
 
-### Writing to Stable Storage
+If your application wants to find any healthy Pod in a StatefulSet,
+and therefore does not need to track each specific Pod,
+you could also connect to the IP address of a `type: ClusterIP` backed
```

Comment:
incomplete sentence ... are you saying that a client can connect to the (Service) backed
by ... ?

Reply:
This sentence looks complete to me.